pages.de/common/babeld.md

```diff
@@ -0,0 +1,24 @@
+# babeld
+
+> Routing-Daemon für Babel, der Firewall-ähnliche Filter benutzt.
+> Weitere Informationen: <https://www.irif.fr/~jch/software/babel/babeld.html>.
+
+- Starte babeld von einer bestimmten Konfigurationsdatei:
```

Comment:
```suggestion
- Starte `babeld` von einer bestimmten Konfigurationsdatei:
```

Reply:
Why should you do it this way? After all, this is not the case in the English version either. Furthermore, I find it a bit strange to write proper names or program names in code blocks.

Reply:
> this is not the case in the English version either

[Wdym?](https://github.com/tldr-pages/tldr/blob/main/pages/common/babeld.md?plain=1#L6)

> I find it a bit strange to write proper names or program names in code blocks

Well, I don't think we do it with proper names anywhere, but for program names it usually makes sense

Reply:
Mhh, I must have overlooked a few. Thanks for your patience and your good suggestions!